src/banner/toast.component.ts

```diff
@@ -50,4 +50,6 @@ export class Toast extends Banner {
 	 *
 	 */
 	@Input() bannerObj: ToastContent;
+
+	@Input() label = "close";
```

Comment:
I did not see your translations PR before doing this. Since this depends on your PR, I'm just going to to wait till its merged and do `[attr.aria-label]="bannerObj.closeLabel"` instead

Reply:
Either way we'll get a merge conflict and have to resolve it. I don't mind which order they go in, but somethings going to collide

Reply:
@cal-smith if you make that DAP fix in this PR, we can disregard/close @youda97's PR and have no conflicts.
Heck, @youda97 can technically do a PR to your branch if you allow it.

Reply:
Oh is this the only DAP fix for banner?

Reply:
I'll pull in the changes then